<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SavingsMenu</class>
 <widget class="QDialog" name="SavingsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Savings Account</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#484848;
color:white;</string>
  </property>
  <widget class="QPushButton" name="deposit_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton#deposit_button {
     background-color: #8080ff;
	font: 14pt &quot;Tahoma&quot;;
	color: white;
    border-style: outset;
    border-width: 2px;
    border-color: black;
 }
 QPushButton#deposit_button:pressed {
     background-color: rgb(0, 0, 0);     
 }
 QPushButton#deposit_button:hover {
     background-color:#6633CC;
 }</string>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdraw_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton#withdraw_button {
     background-color: #8080ff;
	font: 14pt &quot;Tahoma&quot;;
	color: white;
    border-style: outset;
    border-width: 2px;
    border-color: black;
 }
 QPushButton#withdraw_button:pressed {
     background-color: rgb(0, 0, 0);     
 }
 QPushButton#withdraw_button:hover {
     background-color:#6633CC;
 }</string>
   </property>
   <property name="text">
    <string>Withdraw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton#cancel_button {
     background-color: #5725a3;
	font: 14pt &quot;Tahoma&quot;;
	color: white;
    border-style: outset;
    border-width: 2px;
    border-color: black;
 }
 QPushButton#cancel_button:pressed {
     background-color: rgb(0, 0, 0);     
 }
 QPushButton#cancel_button:hover {
     background-color:#5f03ad;
 }</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="interest_rate_label">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="balance_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="interest_rate_label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Interest Rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="interest_rate_label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
